### DISCUSSION TITLE
N150 Wi-Fi Nano USB Adapter sold out, pls recommend other mini dongle

### DISCUSSION THREAD

**Discussion by dingausmwald** [RaspAP/raspap-webgui#979] [Q&A]
This stick is long sold out and is easily mixed up with the v2 version which isnt working out of the box, seems like even the drivers have to be compiled additionally. Please recommend another (mini) usb dongle.

**Comment by billz** [comment]
Discussed here https://docs.raspap.com/faq/#adapters
